python/ray/util/joblib/ray_backend.py

```diff
@@ -43,7 +71,12 @@ def configure(
             n_jobs = ray_cpus
 
         eff_n_jobs = super(RayBackend, self).configure(
-            n_jobs, parallel, prefer, require, **memmappingpool_args
+            n_jobs,
+            parallel,
+            prefer,
+            require,
+            ray_remote_args=ray_remote_args or self.ray_remote_args,
```

Comment:
this check doesn't quite work: if the user passed in `{}`, it will still default to `self.ray_remote_args`

Reply:
Good catch, thanks!